docs/csharp/programming-guide/linq-query-expressions/toc.md

```diff
@@ -1,19 +0,0 @@
-# [LINQ Query Expressions](index.md)
```

Comment:
@mairaw Not ready yet! I see that I didn't edit the TOC correctly. My apologies, it's the first TOC I've edited and this was the first file in the TOC.